addons/delivery/models/delivery_carrier.py

```diff
@@ -108,6 +108,14 @@ def _match_address(self, partner):
             return False
         return True
 
+    def _get_amazon_carrier_name(self):
+        ''' Hook to send a compliant carrier name when needed
+        
+        This function needs to be overridden in the delivery carriers.
+        '''
```

Comment:
The docstring is wrong now =]
This function requires a clean docstring explaining why we need to return `self.field`, and that sometimes it requires an overwrite to return `self.other_field` instead